core/src/replay_stage.rs

```diff
@@ -652,8 +653,14 @@ impl ReplayStage {
             let node_keypair = cluster_info.read().unwrap().keypair.clone();
 
             // Send our last few votes along with the new one
+            let mut last_vote = tower.last_vote();
+            let timestamp_interval =
```

Comment:
seems like we could just pick some hardcoded slot frequency instead of doing this calculation over and over again...